qa/rpc-tests/sc_cert_customfields.py

```diff
@@ -720,9 +777,17 @@ def run_test(self):
         # serialized fe for the proof has 32 byte size, here we use the wrong endianness (version 0 instead of version 1)
         fe1 = get_field_element_with_padding("00110100", 0)
 
-        scProof4 = mcTest.create_test_proof(
-            'sc4', scid4_swapped, epoch_number_4, 10, MBTR_SC_FEE, FT_SC_FEE, epoch_cum_tree_hash_4, constant2, [addr_node1], [bwt_amount],
-            [fe1])
+        scProof4 = mcTest.create_test_proof('sc4',
+                                            scid4_swapped,
+                                            epoch_number_4,
+                                            10,
+                                            MBTR_SC_FEE,
+                                            FT_SC_FEE,
+                                            epoch_cum_tree_hash_4,
+                                            constant = constant2,
```

Comment:
Maybe `constant2` assigned at line 714 (previously defined) could be renamed as `constant4` in order to avoid confusion (being used for parameters preparations related to `sc4`).
Lines involved: 714, 725, 787, 829.
Very minor importance, feel free to ignore.